src/ScrumMaster/Channel/Slack/Channel.php

```diff
@@ -66,7 +66,7 @@ private function postToSlack(Company $company, array $tickets, array $jiraUsersT
 
             $response = $this->slackClient->postToChannel(
                 $this->slackMapping->toSlackId($ticket->assignee()->name()),
-                $this->messageGenerator->forJiraTicket($ticket, $company->companyName())
+                $this->messageGenerator->forJiraTickets([$ticket], $company->companyName())
```

Comment:
Why in the email you pass an array and here you transform the `$thicket` var into an array manually?
I mean, what is the difference between slack and email and why `$ticket` is not already an array?

Reply:
Because this issue (PR) was thinking in a way of pure `Email\Channel`, without modifying (or at least as least at possible) the `Slack\Channel`. 
Actually, I/we might create another PR where we are doing this same feature-idea to the `Slack\Channel` as well. 
BUT in order to keep it simple, I decided to focus this PR only for the `Email\Channel`, because, actually, the `Slack\Channel` is not really hurt by these multiple notifications (aka 1 message per ticket) :)

Reply:
Oh, I got it.
Nice then! >:D